tests/model/test_model.py

```diff
@@ -283,15 +283,15 @@ def test_set_narrow_not_already_set(self, model, initial_narrow, narrow,
             }
         }, {0, 1}),
         ([['stream', 'FOO'],
-         ['topic', 'BOO']], {
+          ['topic', 'BOO']], {
```

Comment:
The fact that `make fix` made formatting changes in files which were _already_ passing the linting tests is concerning. Do we have a plan to enforce `make fix` promptly?

Otherwise, it could potentially accumulate a bunch of unrelated changes in PRs which build on a `master` commit that didn't run `make fix`.

Reply:
I believe the difference is that autopep8 fixes many pep8 rules as specified, but also some additional ones as here: https://pypi.org/project/autopep8/#features

We could add this to CI without `--in-place` and with `--diff`, as a linter. Also autoflake. Ideally this would be part of flake8, so we wouldn't need another checker :/

Reply:
Ah, I see. :/ Nonetheless, we should perhaps enforce this with our CI for consistency.